README.md

```diff
@@ -72,9 +72,8 @@ These scripts are designed to handle all necessary steps, ensuring a seamless se
 
 #### Special note for Windows developers
 
-```
-Ensure that your application and all its dependent libraries are compiled for the same architecture, either all for 32-bit or all for 64-bit. Mixing 32-bit and 64-bit applications and libraries often leads to errors like the one you're seeing.
-```
+> [!IMPORTANT]
```

Comment:
Be aware that this symbology does not work on all interpreters of .md, and as far as I know is kind of github exclusive.  
This is how it looks for me in vscode
![image](https://github.com/user-attachments/assets/174d3963-76f6-4916-9151-bfde5bc4f5f5)

Reply:
Noted, thanks!